packages/main/src/themes/Menu.css

```diff
@@ -25,7 +27,18 @@
 	display: inline-block;
 	vertical-align: middle;
 	padding-inline-start: 0.5rem;
-	float: var(--_ui5-menu_item_icon_float); /* TODO: refactor icon positioning a float and inline-block are incompatible */
+	pointer-events: none;
+	right: var(--_ui5_menu_item_submenu_icon_right);
```

Comment:
If you use [css logical props](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) here, you won't need the [rtl].ui5-menu-item-icon-end styles.

I think [inset-inline-end](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-end) can replace right: var(--_ui5_menu_item_submenu_icon_right);

Reply:
Done.